app/models/application_record.rb

```diff
@@ -3,8 +3,6 @@ class ApplicationRecord < ActiveRecord::Base
 
   before_save :strip_attributes
 
-  DATA_ACCESS_PERIOD_FOR_PUBLISHERS = 1.year.freeze
```

Comment:
DATA_ACCESS_PERIOD_FOR_PUBLISHERS needs to be available in both controllers and models now so I have added it to a data_access_period initializer.